cfgov/v1/migrations/0008_update_tables.py

```diff
@@ -0,0 +1,105 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+from __future__ import print_function
+from itertools import chain
+
+from django.db import migrations
+
+from v1.models.learn_page import LearnPage, DocumentDetailPage
+from v1.models.sublanding_page import SublandingPage
+from v1.models.sublanding_filterable_page import SublandingFilterablePage
+from v1.models.browse_page import BrowsePage
+from v1.models.blog_page import BlogPage
+from v1.models.browse_filterable_page import BrowseFilterablePage
+from v1.models.demo import DemoPage
+from v1.tests.wagtail_pages.helpers import publish_changes
+
+
+def update_table_items(items, wagtail_page):
+    for item in list(filter(
```

Comment:
Another item to consider 
http://www.artima.com/weblogs/viewpost.jsp?thread=98196